models/user.go

```diff
@@ -16,7 +16,7 @@ type User struct {
 	Id       int64  `json:"id"`
 	Username string `json:"username" sql:"not null;unique"`
 	Hash     string `json:"-"`
-	ApiKey   string `json:"-" sql:"not null;unique"`
+	ApiKey   string `json:"api" sql:"not null;unique"`
```

Comment:
As a small nit, could we make this `api_key`?

Reply:
Of course!